automl/google/cloud/automl_v1beta1/tables/gcs_client.py

```diff
@@ -54,18 +54,24 @@ def __init__(self, bucket_name=None, client=None, credentials=None):
                 credentials identify this application to the service. If none
                 are specified, the client will attempt to ascertain the
                 credentials from the environment.
+            project (Optional[string]): The project all future calls will
+                default to. Most methods take `project` as an optional
+                parameter, and can override your choice of `project` supplied
+                here.
+
         """
         if storage is None:
             raise ImportError(_STORAGE_REQUIRED)
 
         if client is not None:
             self.client = client
         elif credentials is not None:
-            self.client = storage.Client(credentials=credentials)
+            self.client = storage.Client(credentials=credentials, project=project)
         else:
             self.client = storage.Client()
 
         self.bucket_name = bucket_name
+        self.project = project
```

Comment:
This attribute is unused.

Reply:
Removed and updated the docstring. Thanks!